### PR TITLE
HTBHF-1560 Content update to match latest prototype.

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

File: src/web/routes/application/enter-dob/enter-dob.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ const pageContent = ({ translate }) => ({
   buttonText: translate('buttons:continue'),
   dayLabel: translate('enterDob.dayLabel'),
   monthLabel: translate('enterDob.monthLabel'),
-  yearLabel: translate('enterDob.yearLabel')
+  yearLabel: translate('enterDob.yearLabel'),
+  whyWeNeedThis: translate('enterDob.whyWeNeedThis')
```
Comment (dwybourn) [inline src/web/routes/application/enter-dob/enter-dob.js]: Open to suggestions of better names.

Comment (mikeSmith08) [reply src/web/routes/application/enter-dob/enter-dob.js]: What about `explanation` or `moreDetail`, I think either would be better.

Comment (YetAnotherMatt) [reply src/web/routes/application/enter-dob/enter-dob.js]: explanation works for me

Comment (dwybourn) [reply src/web/routes/application/enter-dob/enter-dob.js]: Updated to explanation.

File: src/web/views/enter-dob.njk [hunk 2]
```diff
@@ -16,4 +18,9 @@
       text: hint
     }
   }) }}
+
+{{ govukInsetText({
+  text: explanation | safe
+}) }}
```
Comment (gethyn1) [inline src/web/views/enter-dob.njk]: Does this need the `safe` filter?

Comment (dwybourn) [reply src/web/views/enter-dob.njk]: No, removed it.

**Pull request merged**